outputs/http_test.go

```diff
@@ -28,22 +28,35 @@ func TestHTTPClient_Send(t *testing.T) {
 		name           string
 		method         string
 		body           string
+		headers        map[string]string
 		testServerFunc http.HandlerFunc
 		expectedError  string
 	}{
 		{
-			name:   "happy path method get",
-			method: http.MethodGet,
+			name:    "none headers in send method",
+			method:  http.MethodGet,
+			headers: nil,
+			testServerFunc: func(w http.ResponseWriter, r *http.Request) {
+				assert.Equal(t, "bar value", r.Header.Get("fookey"))
+				assert.Equal(t, "", r.Header.Get("POSTEE_EVENT"))
+				fmt.Fprintln(w, "Hello, client")
+			},
+		},
+		{
+			name:    "happy path method get",
+			method:  http.MethodGet,
+			headers: map[string]string{"description": "foo bar baz header"},
```

Comment:
naming is a bit confusing. It's actually not `headers` but result of template evaluation.  It's called `contents`/`data`/`input` in existing implementation of `Send(...)` method